modules/utxo-lib/src/payments/p2tr.ts

```diff
@@ -18,6 +20,19 @@ const BITCOIN_NETWORK = networks.bitcoin;
 const H = Buffer.from('50929b74c1a04954b78b4b6035e97a5e078a5a0f28ec96d547bfee9ace803ac0', 'hex');
 const EMPTY_BUFFER = Buffer.alloc(0);
 
+function isOrdinaryPubkey(pubKey: Uint8Array): boolean {
```

Comment:
the Musig2 BIP uses "Plain" instead of "Ordinary" these days

```suggestion
function isPlainPubkeys(pubKey: Uint8Array): boolean {
```